### PR TITLE
Fix/183

### PR DESCRIPTION
**Pull request by xross**
Addresses issue #183 

Additionally:
- Small improvement to test fail message
- Added --extended test params
- Tidied up some end of test code in the TB

### CODE REVIEW

File: tests/helpers.py [hunk 1]
```diff
@@ -72,10 +68,8 @@ def get_usb_clk_phy(
             "XS1_USB_TERMSEL",
             clk,
             verbose=verbose,
-            test_ctrl=test_ctrl,
             do_timeout=do_timeout,
             complete_fn=complete_fn,
-            expect_loopback=expect_loopback,
             dut_exit_time=dut_exit_time,
         )
```
Comment (kevinyeungxmos) [inline tests/helpers.py:75]: In line 223, 
`if bool(os.getenv("enabletracing")):`  will also be True.

File: tests/usb_phy.py [hunk 2]
```diff
@@ -148,34 +144,10 @@ def end_test(self):
...
-                timeout_time *= 2
-
-                # The events are copied to and from the user application
-                timeout_time *= 2
-
-            self.wait_until(self.xsi.get_time() + timeout_time)
-
-            if self._test_ctrl:
-                # Indicate to the DUT that the test has finished
-                self.xsi.drive_port_pins(self._test_ctrl, 1)
```
Comment (mbanth) [inline tests/usb_phy.py:178]: Is it no longer necessary to drive the port pin?

**Review by mbanth — COMMENTED** [review]
One question

**Review by DaveAtkinsonXMOS — APPROVED** [review]
Bueno

File: tests/helpers.py (hunk 1 above)
Comment (xross) [reply tests/helpers.py:75]: thanks!

File: tests/usb_phy.py (hunk 2 above)
Comment (xross) [reply tests/usb_phy.py:178]: No, we never used this feature.

**Pull request merged**